lesson21/index.js

```diff
@@ -110,5 +112,60 @@ const createTd = (usersData, keys) => {
   return fragment;
 }
 
+const createSortBtn = () => {
+  const sortBtn = createElementWithClassName("button", "sort-btn");
+  sortBtn.dataset.sortStatus = "standard";
+  sortBtn.id = "js-sort-btn";
+  return sortBtn;
+}
+
+const setClickInSortBtn = () => {
+  const standardRows = [...document.querySelector("tbody").querySelectorAll("tr")];
+  const sortBtn = document.getElementById("js-sort-btn");
+
+  sortBtn.addEventListener("click", (e) => {
+    let currentStatus = e.target.dataset.sortStatus;
+    currentStatus = switchSortStatus(currentStatus);
```

Comment:
not current status, you want to say nextStatus

Reply:
I thought you were right.
The status after the switch is not current, but next, which is more appropriate.
549e7a4